docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes_leaderelection-provider.asciidoc

```diff
@@ -23,8 +23,8 @@ providers.kubernetes_leaderelection:
   #leader_renewdeadline: 10
 ----
 
-`enabled`:: (Optional) Defaults to true. To explicitly disable the LeaderElection provider,
-set `enabled: false`.
+`enabled`:: (Optional) Defaults to false. To explicitly enable the LeaderElection provider,
+set `enabled: true`.
```

Comment:
Not true. It will turn on if anywhere in the policy references the `kubernetes_leaderelection`.